dora/src/driver/start.rs

```diff
@@ -103,31 +103,39 @@ pub fn start(content: Option<&str>) -> i32 {
     code
 }
 
+const STDLIB: &[(&str, &str)] = &include!(concat!(env!("OUT_DIR"), "/stdlib.rs"));
+
 fn parse_all_files(vm: &mut VM, ast: &mut Ast, content: Option<&str>) -> Result<(), i32> {
     let fuzzing = content.is_some();
-    let stdlib = vm
+
+    let stdlib_dir = vm
         .args
         .flag_stdlib
-        .clone()
-        .unwrap_or_else(|| "stdlib".to_string());
+        .clone();
 
-    parse_dir(&stdlib, vm, ast).and_then(|_| {
-        if fuzzing {
-            return parse_str(content.unwrap(), vm, ast);
+    if let Some(stdlib) = stdlib_dir {
+        parse_dir(&stdlib, vm, ast)?;
+    } else {
+        for (_, data) in STDLIB {
```

Comment:
we have a tuple of ( filename, content ),
we are not using the filename right now, but i think in the future this is necessary for better error reporting.

Reply:
Yes we should do that. You don't have to do this in this PR though, I can do that later as well if you want.

Reply:
BTW If you want to tackle this, shouldn't be too hard: Reader::from_string should take another argument (the filename).

Reply:
I created a new `Reader::from_bundle` function, do you prefer changing the `Reader::from_string` function?

Reply:
I would actually prefer changing `Reader::from_string` but I would also be fine with your current approach.

Reply:
I think there is no function overloading and also no default arguments in Rust, so I am not sure how the signature should look like:

`Reader::from_string(filename: &str, content: &str)`
`Reader::from_string(filename: Option<&str>, content: &str)`

Reply:
Just always expect the filename.

Reply:
so `Reader::from_string(filename: &str, content: &str)`